examples/starter/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "public_key" {
+  description = "Public API key to authenticate to Atlas"
+}
+variable "private_key" {
+  description = "Private API key to authenticate to Atlas"
+}
+variable "dbuser" {
+  description = "MongoDB Atlas Database User"
+}
+variable "dbuser_password" {
```

Comment:
Sorry I missed this last time!   we've had users create things from examples without editing so I'd leave this "" so they must put a password in, then mention it in the readme.

Reply:
Do you want me to put a default value and leave this as "" or anything else?

Reply:
No default value, just leave as "" and perhaps include a comment that they should enter a password and then a step in the README.  Otherwise people can run it as is, end up with a dbuser_password and have a potential problem if they also open up the access list.   Just to be safe make that impossible.

Reply:
updated variables.tf and added the context in Readme.md as well